github/src/main/kotlin/gropius/sync/github/MongoConfig.kt

```diff
@@ -0,0 +1,37 @@
+package gropius.sync.github
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.core.convert.converter.Converter
+import org.springframework.data.mongodb.core.convert.MongoCustomConversions
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+import java.util.*
+
+/**
+ * Custom configuration for the mongodb driver implementing custom conversion for offsetdatetime
+ */
+@Configuration
+class MongoConfig {
```

Comment:
missing documentation for inner functions / classes

Reply:
f7b2dc1b32aedbf244d06473b330a1c761273056 If you have an idea of the content, please feel welcome to add docs